Libraries/Text/Text.js

```diff
@@ -176,6 +177,13 @@ const Text: React.AbstractComponent<
     default: accessible,
   });
 
+  let flattenedStyle = flattenStyle(style);
+
+  flattenedStyle =
+    typeof flattenedStyle?.fontWeight === 'number'
+      ? {...flattenedStyle, fontWeight: flattenedStyle?.fontWeight.toString()}
```

Comment:
I don't think we need to create a new object and could just replace the `fontWeight` value.

Reply:
Yes, made changes.